lib/test/Test/Pos/Cbor/RefImpl.hs

```diff
@@ -150,23 +147,22 @@ data AdditionalInformation =
   deriving (Eq, Show)
 
 instance Arbitrary UInt where
-  arbitrary =
-    sized $ \n ->
-      oneof $ take (1 + n `div` 2)
-        [ UIntSmall <$> choose (0, 23)
-        , UInt8     <$> arbitraryBoundedIntegral
-        , UInt16    <$> arbitraryBoundedIntegral
-        , UInt32    <$> arbitraryBoundedIntegral
-        , UInt64    <$> arbitraryBoundedIntegral
-        ]
+    arbitrary = sized $ \n ->
+        oneof $ take (1 + n `div` 2)
+          [ UIntSmall <$> choose (0, 23)
+          , UInt8     <$> arbitraryBoundedIntegral
+          , UInt16    <$> arbitraryBoundedIntegral
+          , UInt32    <$> arbitraryBoundedIntegral
+          , UInt64    <$> arbitraryBoundedIntegral
+          ]
 
 instance Arbitrary AdditionalInformation where
-  arbitrary =
-    frequency
-      [ (7, AiValue <$> arbitrary)
-      , (2, pure AiIndefLen)
-      , (1, AiReserved <$> choose (28, 30))
-      ]
+    arbitrary =
+        frequency
+          [ (7, AiValue <$> arbitrary)
```

Comment:
Perhaps this is useless, but it'd be good to have a comment explaining why this distribution was chosen (i.e. to create more interesting tests).